### ISSUE TITLE
Segmentation fault (core dumped) in cuda 11.6

### ISSUE THREAD

**Issue by vansin** [open-mmlab/mmyolo#88]
Copy from WeChat Group

![09c65f487b5ac602330476af2ce6bb2](https://user-images.githubusercontent.com/25839884/192154394-161eb89e-5d4d-4313-940a-ce45411b0481.jpg)

![46abbb8af2e6488cb971fa0d88adf9a](https://user-images.githubusercontent.com/25839884/192154461-801db796-7cff-48a9-b312-ab28c479b545.jpg)

![image](https://user-images.githubusercontent.com/25839884/192154488-b563cbf7-63a4-498e-8861-15249837e24b.png)

**Comment by PeterH0323** [comment]
Hi
What is the version of the pytorch?

please show the pip list

**Comment by lincaiming** [comment]
pip list
addict             2.4.0
albumentations     1.0.1
appdirs            1.4.4
attrs              22.1.0
brotlipy           0.7.0
certifi            2022.9.24
cffi               1.14.6
charset-normalizer 2.1.1
cityscapesScripts  2.2.0
click              8.1.3
colorama           0.4.5
coloredlogs        15.0.1
commonmark         0.9.1
contourpy          1.0.5
coverage           6.4.4
cryptography       37.0.2
cycler             0.11.0
Cython             0.29.32
flatbuffers        2.0.7
fonttools          4.37.3
humanfriendly      10.0
idna               3.4
imagecorruptions   1.1.2
imageio            2.22.0
importlib-metadata 4.12.0
iniconfig          1.1.1
joblib             1.2.0
kiwisolver         1.4.4
lmdb               1.3.0
Markdown           3.4.1
matplotlib         3.6.0
mkl-fft            1.3.1
mkl-random         1.2.2
mkl-service        2.4.0
mmcv               2.0.0rc1
mmdet              3.0.0rc0  /home/ming/game/mmlab/mmdetection
mmengine           0.1.0     /home/ming/game/mmlab/mmengine
mmyolo             0.1.0     /home/ming/game/mmlab/mmyolo
model-index        0.1.11
mpmath             1.2.1
networkx           2.8.6
ninja              1.10.2.3
numpy              1.23.1
olefile            0.46
onnx               1.7.0
onnxoptimizer      0.3.1
onnxruntime        1.12.1
opencv-python      4.6.0.66
openmim            0.3.2
ordered-set        4.1.0
packaging          21.3
pandas             1.5.0
Pillow             9.2.0
pip                22.1.2
pluggy             1.0.0
protobuf           3.19.5
py                 1.11.0
pycocotools        2.0.5
pycparser          2.21
Pygments           2.13.0
pyOpenSSL          22.0.0
pyparsing          3.0.9
pyquaternion       0.9.9
PySocks            1.7.1
pytest             7.1.3
pytest-runner      6.0.0
python-dateutil    2.8.2
PyTurboJPEG        1.6.7
pytz               2022.2.1
PyWavelets         1.4.1
PyYAML             6.0
requests           2.28.1
rich               12.5.1
scikit-image       0.19.3
scikit-learn       1.1.2
scipy              1.9.1
setuptools         63.4.1
six                1.16.0
sklearn            0.0
sympy              1.11.1
tabulate           0.8.10
termcolor          2.0.1
terminaltables     3.1.10
threadpoolctl      3.1.0
tifffile           2022.8.12
timm               0.6.7
tomli              2.0.1
torch              1.12.1
torchaudio         0.12.1
torchvision        0.13.1
tqdm               4.64.1
typing             3.7.4.3
typing_extensions  4.3.0
urllib3            1.26.11
wheel              0.37.1
yapf               0.32.0
zipp               3.8.1

**Comment by lincaiming** [comment]
conda list

_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
addict                    2.4.0                    pypi_0    pypi
albumentations            1.0.1                    pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
attrs                     22.1.0                   pypi_0    pypi
blas                      1.0                         mkl  
brotlipy                  0.7.0           py38h0a891b7_1004    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
ca-certificates           2022.9.24            ha878542_0    conda-forge
certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
cffi                      1.14.6           py38ha65f79e_0    conda-forge
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
cityscapesscripts         2.2.0                    pypi_0    pypi
click                     8.1.3                    pypi_0    pypi
colorama                  0.4.5                    pypi_0    pypi
coloredlogs               15.0.1                   pypi_0    pypi
commonmark                0.9.1                    pypi_0    pypi
contourpy                 1.0.5                    pypi_0    pypi
coverage                  6.4.4                    pypi_0    pypi
cryptography              37.0.2           py38h2b5fc30_0    conda-forge
cudatoolkit               11.6.0              hecad31d_10    conda-forge
cycler                    0.11.0                   pypi_0    pypi
cython                    0.29.32                  pypi_0    pypi
ffmpeg                    4.3                  hf484d3e_0    pytorch
flatbuffers               2.0.7                    pypi_0    pypi
fonttools                 4.37.3                   pypi_0    pypi
freetype                  2.10.4               h0708190_1    conda-forge
gmp                       6.2.1                h58526e2_0    conda-forge
gnutls                    3.6.13               h85f3911_1    conda-forge
humanfriendly             10.0                     pypi_0    pypi
idna                      3.4                pyhd8ed1ab_0    conda-forge
imagecorruptions          1.1.2                    pypi_0    pypi
imageio                   2.22.0                   pypi_0    pypi
importlib-metadata        4.12.0                   pypi_0    pypi
iniconfig                 1.1.1                    pypi_0    pypi
intel-openmp              2021.4.0          h06a4308_3561  
joblib                    1.2.0                    pypi_0    pypi
jpeg                      9e                   h166bdaf_1    conda-forge
kiwisolver                1.4.4                    pypi_0    pypi
lame                      3.100             h7f98852_1001    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.38                 h1181459_1  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libiconv                  1.17                 h166bdaf_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libstdcxx-ng              11.2.0               h1234567_1  
libtiff                   4.2.0                hf544144_3    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
lmdb                      1.3.0                    pypi_0    pypi
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
markdown                  3.4.1                    pypi_0    pypi
matplotlib                3.6.0                    pypi_0    pypi
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0            py38h95df7f1_0    conda-forge
mkl_fft                   1.3.1            py38h8666266_1    conda-forge
mkl_random                1.2.2            py38h1abd341_0    conda-forge
mmcv                      2.0.0rc1                 pypi_0    pypi
mmdet                     3.0.0rc0                  dev_0    <develop>
mmengine                  0.1.0                     dev_0    <develop>
mmyolo                    0.1.0                     dev_0    <develop>
model-index               0.1.11                   pypi_0    pypi
mpmath                    1.2.1                    pypi_0    pypi
ncurses                   6.3                  h5eee18b_3  
nettle                    3.6                  he412f7d_0    conda-forge
networkx                  2.8.6                    pypi_0    pypi
ninja                     1.10.2.3                 pypi_0    pypi
numpy                     1.23.1           py38h6c91a56_0  
numpy-base                1.23.1           py38ha15fc14_0  
olefile                   0.46               pyh9f0ad1d_1    conda-forge
onnx                      1.7.0                    pypi_0    pypi
onnxoptimizer             0.3.1                    pypi_0    pypi
onnxruntime               1.12.1                   pypi_0    pypi
opencv-python             4.6.0.66                 pypi_0    pypi
openh264                  2.1.1                h780b84a_0    conda-forge
openjpeg                  2.4.0                hb52868f_1    conda-forge
openmim                   0.3.2                    pypi_0    pypi
openssl                   1.1.1o               h166bdaf_0    conda-forge
ordered-set               4.1.0                    pypi_0    pypi
packaging                 21.3                     pypi_0    pypi
pandas                    1.5.0                    pypi_0    pypi
pillow                    9.2.0                    pypi_0    pypi
pip                       22.1.2           py38h06a4308_0  
pluggy                    1.0.0                    pypi_0    pypi
protobuf                  3.19.5                   pypi_0    pypi
py                        1.11.0                   pypi_0    pypi
pycocotools               2.0.5                    pypi_0    pypi
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygments                  2.13.0                   pypi_0    pypi
pyopenssl                 22.0.0             pyhd8ed1ab_1    conda-forge
pyparsing                 3.0.9                    pypi_0    pypi
pyquaternion              0.9.9                    pypi_0    pypi
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pytest                    7.1.3                    pypi_0    pypi
pytest-runner             6.0.0                    pypi_0    pypi
python                    3.8.13               h12debd9_0  
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.8                      2_cp38    conda-forge
pytorch                   1.12.1          py3.8_cuda11.6_cudnn8.3.2_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pyturbojpeg               1.6.7                    pypi_0    pypi
pytz                      2022.2.1                 pypi_0    pypi
pywavelets                1.4.1                    pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
readline                  8.1.2                h7f8727e_1  
requests                  2.28.1             pyhd8ed1ab_1    conda-forge
rich                      12.5.1                   pypi_0    pypi
scikit-image              0.19.3                   pypi_0    pypi
scikit-learn              1.1.2                    pypi_0    pypi
scipy                     1.9.1                    pypi_0    pypi
setuptools                63.4.1           py38h06a4308_0  
six                       1.16.0             pyh6c4a22f_0    conda-forge
sklearn                   0.0                      pypi_0    pypi
sqlite                    3.39.2               h5082296_0  
sympy                     1.11.1                   pypi_0    pypi
tabulate                  0.8.10                   pypi_0    pypi
termcolor                 2.0.1                    pypi_0    pypi
terminaltables            3.1.10                   pypi_0    pypi
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2022.8.12                pypi_0    pypi
timm                      0.6.7                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0  
tomli                     2.0.1                    pypi_0    pypi
torchaudio                0.12.1               py38_cu116    pytorch
torchvision               0.13.1               py38_cu116    pytorch
tqdm                      4.64.1                   pypi_0    pypi
typing                    3.7.4.3                  pypi_0    pypi
typing_extensions         4.3.0              pyha770c72_0    conda-forge
urllib3                   1.26.11            pyhd8ed1ab_0    conda-forge
wheel                     0.37.1             pyhd3eb1b0_0  
xz                        5.2.5                h7f8727e_1  
yapf                      0.32.0                   pypi_0    pypi
zipp                      3.8.1                    pypi_0    pypi
zlib                      1.2.12               h5eee18b_3  
zstd                      1.5.0                ha95c52a_0    conda-forge

**Comment by PeterH0323** [comment]
Hi @lincaiming 

I think this is about the version of cudatoolkit , I suggest you install cudatoolkit 11.3:

```bash 
conda install pytorch==1.11.0 torchvision==0.12.0 torchaudio==0.11.0 cudatoolkit=11.3 -c pytorch
```

**Comment by cydiachen** [comment]
![image](https://user-images.githubusercontent.com/26337720/192188350-4859f1d8-9c45-4066-85c8-bf2cc6533a26.png)
It seems that the error is raised when loading coco-like annotations, May I ask how large is your dataset and how big is your host memory (RAM not GPU memory).

**Comment by hhaAndroid** [comment]
No response after time
